ccpage171.py

```diff
@@ -0,0 +1,106 @@
+        restaurant_details = self.restuaruant_name.title() + " serves " + self.restuaruant_cuisine.title() + " food."
+        return restaurant_details
+
+    # Add a method so I can set the number of guests served
+    def set_number_served(self, guests):
+        """Determine the number of people served at the restaurant"""
+        self.number_served = guests
+
+    # Add a method to increment the total number of people served
+    def increment_number_served(self, incremented_guests):
+        """Increment the total number of guests served"""
+        self.number_served += incremented_guests
+
+    # I added a method here so I can call the method, instead of printing a statment about the total number of guests served
+    def statement_for_number_served(self):
+        """Print a statement that tells the number of people served"""
+        print(f"The total number of people served is {int(self.number_served)}.")
+
+restaurant = Restaurant('grotto', 'italian')
+print(restaurant.describe_restaruant())
+print(f"{restaurant.restuaruant_name.title()} has served {restaurant.number_served} people.")
+
+print('\n')
+
+restaurant.number_served = 120
+restaurant.statement_for_number_served()
+
+print('\n')
+
+restaurant.set_number_served(200)
+restaurant.statement_for_number_served()
+
+print('\n')
+
+restaurant.increment_number_served(25)
+restaurant.statement_for_number_served()
+
+print('\n')
+
+restaurant.increment_number_served(25)
+restaurant.statement_for_number_served()
+
+
+# Add an attribute called login_attempts to your User class from Exercise 9-3 (page 166).
+# Write a method called increment_ login_attempts() that increments the value of login_attempts by 1.
+# Write another method called reset_login_attempts() that resets the value of login_ attempts to 0.
+# Make an instance of the User class and call increment_login_attempts() several times.
+# Print the value of login_attempts to make sure it was incremented properly, and then call reset_login_attempts().
+
+class User():
+    """An attempt to model a user"""
+    def __init__(self, first, last):
+        """Initialize the attributes"""
+        self.first = first
+        self.last = last
+        self.login_attempts = 0
+
+    def increment_login_attempts(self, increase_attempts):
+        """Increment the login attempts"""
+        self.login_attempts += increase_attempts
+    
+    def reset_login_attempts(self, reset):
+        """Reset the login attempts to 0"""
+        self.login_attempts = reset
+    
+    def print_total_login_attempts(self):
+        """Print a statement with the number of login attempts"""
+        if self.login_attempts >1: 
+            print(f"The total attempts are: {self.login_attempts}.")
+        else:
+            print(f"The total attempt is: {self.login_attempts}.")
+
+user = User('amy', 'haddad')
+user.increment_login_attempts(1)
+user.increment_login_attempts(1)
+user.increment_login_attempts(1)
+user.increment_login_attempts(1)
```

Comment:
@paulghaddad Is this the best way to increment by 1? Is there another way to do this automatically? I tried using a variable in the method that was equal to 0, but I got an error. Here's what I did:

```
def increment_login_attempts(self, increase_attempts):
        """Increment the login attempts"""
        attempts = 1
        self.login_attempts += attempts
```
Because I got an error, I resorted the the longer, manual version of incrementing a number. Thoughts on this?